support-utils/cre_astro_software_gather_logs.sh

```diff
@@ -0,0 +1,392 @@
+#    export POD=$POD;echo $POD pod is in bad state;echo "Collecting Describe output of bad state pod $POD in $NS Namespace ";kubectl describe pod $POD  > "$NS/$POD-BAD_$NS.log" -n $NS   
+#    done
+#done
+
+
+
+
+
+
+# echo "======================Gathering Describe output of Bad state pod======================"
+# for NS in $(kubectl get ns --no-headers| awk '{print $1}'); 
+# do
+#   for BAD_POD in $(kubectl get pods --no-headers -n $NS |grep -v Running|grep -v Completed|awk '{ print $1}') ; do
+#     export BAD_POD=$BAD_POD;echo $BAD_POD pod is in bad state;echo "Collecting Describe output of bad state pod $BAD_POD in $NS Namespace ";kubectl describe pod $BAD_POD  > "$NS/BadPods/$BAD_POD-BAD_$NS.log" -n $NS   
+#     done
+# done
+
+
+# echo "======================Gathering log output of Bad state pod======================"
+# for NS in $(kubectl get ns --no-headers| awk '{print $1}'); 
+# do
+#   for BAD_POD in $(kubectl get pods --no-headers -n $NS |grep -v Running|grep -v Completed|awk '{ print $1}') ; do
+#     export BAD_POD=$BAD_POD;echo "Starting to Collect log of the bad state pod $BAD_POD in $NS namespace";for container_name in $(kubectl get pods $BAD_POD -o jsonpath='{.spec.containers[*].name}' -n $NS|awk '{NF-=0; OFS="\n"; $1=$1}1' | sort);do
+#     echo Collecting log of the container $container_name in pod $BAD_POD in the $NS namespace now;kubectl logs $BAD_POD -n $NS -c $container_name > "$NS/BadPods/$BAD_POD-pod_$container_name-BADPODLOG.log"  
+#     done
+# done
+# done
+
+
+
+
+# echo "======================Gathering logs of All the pods ======================"
+# for NS in $(kubectl get ns --no-headers| awk '{print $1}'); 
+# do
+#   for POD in $(kubectl get pods --no-headers -n $NS |awk '{ print $1}') ; do
+#     export POD=$POD;echo "Starting to Collect log of the pod $POD in $NS namespace";for container_name in $(kubectl get pods $POD -o jsonpath='{.spec.containers[*].name}' -n $NS|awk '{NF-=0; OFS="\n"; $1=$1}1' | sort);do
+#     echo Collecting log of the container $container_name in pod $POD in the $NS namespace now;kubectl logs $POD -n $NS -c $container_name > "$NS/AllPodlogs/$POD-pod_$container_name-container.log"  
+#     done
+# done
+# done
+
+
+
+
+#####====================================================================================================================================================#####
+# ####Gathering Describe output of all the Nodes
+# echo "======================Gathering Describe output of all the nodes======================"
+# for NODE in $(kubectl get nodes --no-headers |awk '{ print $1}') ; do
+#     echo "Collecting Describe output of Node $NODE ";kubectl describe nodes $NODE > "$ASTRONOMER_NAMESPACE/Deployment_logs/DESCRIBE_$NODE.log"
+# done
+#####==================================================================================================================================================#####
```

Comment:
These comment blocks should be removed or put behind an ENV var or command line arg.